File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/FieldStandardTime/FieldStandardTimeCreateDto.cs
namespace OregonTilth.Models.DataTransferObjects
{
    public class FieldStandardTimeCreateDto
    {
        public int WorkbookID { get; set; }
        public int FieldLaborActivityID { get; set; }
        public int LaborTypeID { get; set; }
        public int? MachineryID { get; set; }
        public int? FieldUnitTypeID { get; set; }
    }
}

File: OregonTilth.Web/src/app/shared/models/forms/field-standard-times/field-standard-time-create-dto.ts
import { FieldLaborActivityDto } from "../../generated/field-labor-activity-dto";
import { FieldUnitTypeDto } from "../../generated/field-unit-type-dto";
import { LaborTypeDto } from "../../generated/labor-type-dto";
import { MachineryDto } from "../../generated/machinery-dto";

export class FieldStandardTimeCreateDto {
    WorkbookID: number;
    FieldLaborActivityID: number;
    FieldLaborActivity: FieldLaborActivityDto;
    LaborTypeID: number;
    LaborType: LaborTypeDto;
    MachineryID: number;
    Machinery: MachineryDto;
    FieldUnitTypeID: number;
    FieldUnitType: FieldUnitTypeDto;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}
